src/Metrics.Integrations.Linters/Linters/ArgBuilder.cs

```diff
@@ -46,13 +49,24 @@ private bool IsInclude(object value)
                 .Case<bool>(v => isInclude = v != BoolDefault)
                 .Case<string>(v => isInclude = !string.IsNullOrEmpty(v) && v != StringDefault)
                 .Case<object>(v => isInclude = v != ObjectDefault)
+                .Case<List<string>>(v => isInclude = v != StringListDefault && v.Count != 0)
                 .Default(v => isInclude = false);
 
             return isInclude;
         }
 
         private static string BuildArgument(KeyValuePair<ArgAttribute, object> arg)
         {
+            // check if value is List<string>
+            if (arg.Value != null && arg.Value is IList)
```

Comment:
You should check that value is the collection of strings, e.g. `is List<string>`.